test/Microsoft.TestPlatform.AcceptanceTests/ExecutionTests.cs

```diff
@@ -216,6 +223,7 @@ public void IncompatiableSourcesWarningShouldBeDisplayedInTheConsole(RunnerInfo
             this.InvokeVsTest(arguments);
 
             this.ValidateSummaryStatus(1, 1, 1);
+            this.ExitCodeEquals(1); // there are failing tests
```

Comment:
When one dll is picked for execution and other is ignored, exit code will be 1 ?

Reply:
Exit code is 1 because there is failing test.